### DISCUSSION TITLE
Program structure

### DISCUSSION THREAD

**Discussion by MaartenGoris** [ArminJo/JK-BMSToPylontechCAN#9] [General]
Hi,

I have written smaller programs and i use arduino.

```
void setup() {
  // put your setup code here, to run once:
  int result = myFunction(2, 3);
}

void loop() {
  // put your main code here, to run repeatedly:
}

```
However i'm having troubles finding the entry point here to understand what is going on.

I'm using platform IO on VS code.

Can you please set me on track on how to download this program?

Thanks

**Comment by ArminJo** [comment]
Look [here](https://github.com/ArminJo/JK-BMSToPylontechCAN/blob/f28ccb3e13e4221ee93fc649912816ca67619ac6/JK-BMSToPylontechCAN/JK-BMSToPylontechCAN.ino#L344).
